modules/installation-special-config-encrypt-disk-tpm2.adoc

```diff
@@ -4,43 +4,47 @@
 
 [id="installation-special-config-encrypt-disk-tpm2_{context}"]
 = Enabling TPM v2 disk encryption
-Use this procedure to enable TPM v2 mode disk encryption during {product-title} deployment.
+Use this procedure to enable TPM v2 mode disk encryption during an {product-title} installation.
 
 [NOTE]
 ====
 On previous versions of {op-system}, disk encryption was configured by specifying `/etc/clevis.json` in the Ignition config. That file is not supported in clusters created with {product-title} 4.7 or above, and the LUKS device should be configured directly via the Ignition `luks` section.
 ====
 
+.Prerequisites
+
+* You downloaded the {product-title} installation program on your installation node.
```

Comment:
```suggestion
* You have downloaded the {product-title} installation program on your installation node.
```

Just for consistency with the bullet below.

Reply:
Resolved.